lunchtime12/lunchtime12.ipynb

```diff
@@ -0,0 +1,757 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Welcome to Dash\n",
+    "\n",
+    "UI elements not quite as messy as ipywidgets.\n",
+    "Slide Type\n",
+    "\n",
+    "Dash is a comparably easy and stable way to build standalone or jupyter based widgets/apps.\n",
+    "\n",
+    "All components listed here can also be found under https://dash.plotly.com/dash-core-components for a more specific documentation\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "notes"
+    }
+   },
+   "source": [
+    "Simple text is automatically shown side by side: <br>\n",
```

Comment:
this comes a bit abruptly, maybe something general about the workings of Dash, that it is on top of Flask etc and that here you use the div component to interact